Readme.md

```diff
@@ -82,6 +82,40 @@ if (true)
 
 Also, notice the use of whitespace before and after the condition statement.
 
+## Pass all optional parameters as a single object
+
+Pass all optional parameters as a single object, it makes it easier to change function
+definitions.
+
+*Right:*
+
+```js
+function foo(bar, opts) {
+  this.color = opts.color;
```

Comment:
Recommend using _.defaults to both ensures defaults, and documents it in the code with strong locality.

opts = _.defaults(opts, {
    colour: 'red',
    speed: '250m.s-1',
    sound: 'wooo'
  })